server/src/main/java/org/opensearch/rest/ReplacedNamedRoute.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.rest;
+
+import org.opensearch.OpenSearchException;
+import org.opensearch.transport.TransportService;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * A replaced named Route
+ *
+ * @opensearch.internal
+ */
+public class ReplacedNamedRoute extends RestHandler.ReplacedRoute implements NamedRouteWrapper {
```

Comment:
Lets tackle this and deprecated route separately in another PR.

Reply:
👍 added an item to meta